src/Reactor.cc

```diff
@@ -399,11 +399,6 @@ vector<string> Reactor::getSetups()
 			}
 		});
 	}
```

Comment:
Shouldn't we also remove this for loop? Because now we still search in 2 directories (also the current directory).

Reply:
What's then the proper way to get only the specific path? (I thought the context stuff was the proper way to get these paths...)